package/cfg/k3s-cis-1.7-hardened/master.yaml

```diff
@@ -657,11 +666,13 @@ groups:
           on the control plane node and set the --audit-log-maxbackup parameter to 10 or to an appropriate
           value. For example,
           --audit-log-maxbackup=10
+          Permissive.
```

Comment:
Your last commit added many skips and `Permissive` to the hardened tests. Is that intentional?

Reply:
Regarding the `Permissive.`, this is a way to differentiate `Not Applicable `skips from `Permissive` skips. As for the increase of skip numbers, they are aligned with the K3s Hardening Guide and previous profiles.